grpc-protocol/src/main/java/com/linecorp/armeria/common/grpc/protocol/Base64Decoder.java

```diff
@@ -125,17 +123,11 @@ private int decodedBufferSize(int len) {
     @Override
     public boolean process(byte value) throws Exception {
         final byte decodedByte = DECODABET[value & 0xFF];
-        if (decodedByte < WHITE_SPACE_ENC) {
+        if (decodedByte <= WHITE_SPACE_ENC) {
```

Comment:
Could be `decodedByte < EQUALS_SIGN_ENC` ?

Reply:
Fixed. 😉